CPT331.iOS/CPT331.iOS/EventManager.swift

```diff
@@ -62,14 +74,16 @@ class EventManager {
                     let name = event["name"].string
```

Comment:
`let name = event[EventField.name.rawValue].string` 
This would be valid syntax, but it gets quite cumbersome at that point. I'd have to have to use "magic strings" regardless of what enums i have, for example: 
`event["point"]["lng"].double`
What would i do with "lng"?

Tbh, i think it might be less confusing to just remove the enum all together.